<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="DEVICE_COMBO">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>751</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="MyLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>741</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(100, 149, 200)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>50</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>初始化</string>
    </property>
    <widget class="QPushButton" name="ENUM_BUTTON">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查找设备</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OPEN_BUTTON">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开设备</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CLOSE_BUTTON">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭设备</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>160</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>图像采集</string>
    </property>
    <widget class="QRadioButton" name="CONTINUS_MODE_RADIO">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>连续模式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="TRIGGER_MODE_RADIO">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>触发模式</string>
     </property>
    </widget>
    <widget class="QPushButton" name="START_GRABBING_BUTTON">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开始采集</string>
     </property>
    </widget>
    <widget class="QPushButton" name="STOP_GRABBING_BUTTON">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>停止采集</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="SOFTWARE_TRIGGER_CHECK">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>软触发</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SOFTWARE_ONCE_BUTTON">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>轻触发一次</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>300</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>图像保存</string>
    </property>
    <widget class="QPushButton" name="SAVE_BMP_BUTTON">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存BMP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SAVE_JPG_BUTTON">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存JPG</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>410</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>图像处理</string>
    </property>
    <widget class="QPushButton" name="DETECT_LINE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>直线检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DETECT_CIRCLE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>圆检测</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
